specs/TrackingPrevention.md

```diff
@@ -0,0 +1,206 @@
+Tracking Prevention
+===
+
+# Background
+The WebView2 team has been asked for an API to toggle tracking prevention and also ability to control levels
+of tracking prevention.
+
+We are proposing two API's
+IsTrackingPreventionEnabled: This API allows you to enable/disable the tracking prevention feature. By default
+tracking prevention feature is enabled and set to `Balanced` level for WebView2. You can set the
+`CoreWebView2EnvironmentOptions.IsTrackingPreventionEnabled` property to false to disable the tracking prevention
+feature for WebView2 before creating environment that also skips the related code and improves the performance.
+
+TrackingPreventionUserPreference: This API allows you to control levels of tracking prevention for WebView2 which
+are associated with a profile when the feature is enabled and otherwise TrackingPreventionUserPreference is
+CoreWebView2TrackingPreventionUserPreferenceKind.Off and can't be changed to other kinds.
+The levels are similar to Edge: `Off`, `Basic`, `Balanced` and `Strict`.
+
+For reference, in the screenshot below, this API sets the levels of tracking prevention as a WebView2 API.
+
+![Edge Tracking Prevention](images/TrackingPreventionLevels.png)
+
+# Examples
+## IsTrackingPreventionEnabled
+
+```c#
+/// Create WebView Environment with option that disable tracking prevention feature.
+
+void CreateEnvironmentWithOption()
+{
+    CoreWebView2EnvironmentOptions options = new CoreWebView2EnvironmentOptions();
+    // If we're displaying app content or otherwise are not worried about tracking, we can
+    // disable the tracking prevention feature to improve runtime performance.
+    options.IsTrackingPreventionEnabled = false;
+    CoreWebView2Environment environment = await CoreWebView2Environment.CreateAsync(BrowserExecutableFolder, UserDataFolder, options);
+}
+```
+
+```cpp
+Microsoft::WRL::ComPtr<ICoreWebView2EnvironmentOptions3> options3;
+if (options.As(&options3) == S_OK)
+{
+    // If we're displaying app content or otherwise are not worried about tracking, we can
+    // disable the tracking prevention feature to improve runtime performance.
+    CHECK_FAILURE(options3->put_IsTrackingPreventionEnabled(FALSE));
+}
+```
+
+## TrackingPreventionUserPreference
+/// Example to set level of tracking preference.
```

Comment:
No need for /// comment. Also would be good to elaborate just slightly on what the example is doing. Having just code that sets a property isn't very helpful for a sample but if the only sample we have for this property is the user changing the value through UI then this seems fine. We can say that here:

```suggestion
Example of code in end user facing UI to change the TrackingPreventionLevel.
```